src/libraries/System.Security.Cryptography/tests/Sha384Tests.cs

```diff
@@ -42,7 +118,7 @@ public void Sha384_NistShaAll_1()
         }
 
         [Fact]
-        public void Sha256_Fips180_MultiBlock()
+        public void SHA384_Fips180_MultiBlock()
```

Comment:
Why change the casing here?  All of the rest of the methods are little-H/little-A

Reply:
Ah, well, it was called out as SHA**256** instead of SHA**384** and I guess my fingers just did the wrong thing while fixing the algorithm size. Will fix.

Reply:
Ah.  Clearly my internal diffing algorithm stopped at the first change. :)